packages/select/src/components/omnibar/omnibar.tsx

```diff
@@ -21,7 +21,7 @@ import {
     Utils,
 } from "@blueprintjs/core";
 
-import * as Classes from "../../common/classes";
+import * as Classes from "../classes";
```

Comment:
I think the previous path was fine...? a `common/` folder next to `components/`?

Reply:
there's nothing else in common; do we need another directory level?

Reply:
it's just nice to have the convention across packages